vignettes/texreg.Rnw

```diff
@@ -439,16 +439,18 @@ Class			& Package			& Description \\
 \bottomrule
 \endfoot
 
-\caption[List of 96 supported model types (version 1.37).]{\label{tab:types} List of 96 supported model types (version 1.37).} \\
+\caption[List of 99 supported model types (version 1.36.24).]{\label{tab:types} List of 99 supported model types (version 1.36.24).} \\
 \endlastfoot
-`\code{aftreg}'			& \pkg{eha}			& Accelerated failure time regression \\
+
+`\code{aftreg}'			& \pkg{eha}			    & Accelerated failure time regression \\
```

Comment:
Can you check the spacing in this table please?

Reply:
yeah, sorry, I did it already, but as I resolved conflicts it got messed up again. I'll do it for the second time